packages/neuron-wallet/src/models/transaction-fee.ts

```diff
@@ -0,0 +1,11 @@
+export default class TransactionFee {
```

Comment:
It seems merged in the develop branch, please rebase on it first.

Reply:
It's a draft PR and didn't request review yet .....